src/components/entries/FEEL/FeelPopup.js

```diff
@@ -200,7 +209,22 @@ function FeelPopupComponent(props) {
       <Popup.Title
         title={ title }
         emit={ emit }
-        draggable />
+        draggable>
+        {type === 'feel' && (
+          <a href="https://docs.camunda.io/docs/components/modeler/feel/what-is-feel/" target="_blank" class="bio-properties-panel-feel-popup__title-link">
```

Comment:
I wonder if we want to make this pluggable, or make the popup editor a Camunda-only component (cf. https://github.com/bpmn-io/internal-docs/issues/825). 

To me it does not really make sense to embed such a link inside an agnostic library. Happy to compromise temporarily, of course.

Reply:
I agree, this should not be in an agnostic library. We talked about moving the popout editor into its own component anyway, likely when we want to add more FEEL guidance/data awareness to it. 
So I would not spend the time right now to make the header pluggable

Reply:
Then let's keep it as is (we compromise) for the moment.